### PR TITLE
feat: no hardcoded env

### PR DESCRIPTION
**Pull request by grixu**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: build-frontend/action.yml [hunk 1]
```diff
@@ -34,45 +31,32 @@ inputs:
     description: 'NPM registry'
     required: false
     default: 'https://registrynpm.storefrontcloud.io'
-  api_base_url:
-    description: 'Base Middleware API URL for Storefront. By default https://project_name.region.gcp.storefrontcloud.io/api'
-    required: false
-  multistore_enabled:
```
Comment (Szymon-dziewonski) [inline build-frontend/action.yml:40]: out of curiosity, how those env will be passed then into docker containers?

Comment (grixu) [reply build-frontend/action.yml:40]: All will be set in Console ;)

File: README.md [hunk 2]
```diff
@@ -20,45 +20,27 @@ with:
...
-- `image_provider`: Switch the image provider on the Nuxt frontend. More details in the [Nuxt image docs](https://image.nuxt.com/get-started/providers). Know the differences between the Nuxt & Next image modules and their configuration. Optional field. The default value is undefined because Nuxt automatically falls back to `ipx`
-- `image_provider_upload_url`: URL for uploading images to the selected image provider (e.g., Cloudinary). Do not set this value if you are not using any image provider. Any value here will attempt to run the module in the frontend code, resulting in an error and frontend unavailability. Optional field.
-- `image_provider_fetch_url`: URL for fetching images from the selected image provider. Optional field.
-- `coveo_organization_id`: Coveo organization ID required for coveo integration. Optional field.
-- `coveo_access_token`: Coveo organization access token required for coveo integration. Optional field.
-- `coveo_tracking_id`: Coveo tracking id required for coveo integration. Optional field.
```
Comment (FRSgit) [inline README.md:59]: Suggestion:
wondering whether we shouldn't add those docs somewhere to our readme. Like: these are env variables, that are set by the console automatically: {list_of_variables}. To change them, go to console admin panel.

**Review by FRSgit — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (grixu) [reply README.md:59]: Good idea 👍

**Pull request merged**